### DISCUSSION TITLE
Bundle size optimization

### DISCUSSION THREAD

**Discussion by Guarn** [heroui-inc/heroui#611] [Q&A]
Hey,

I'm trying to reduce bundle size with vite@react, and, there is something I don't understand.
I added rollup-plugin-visualizer to plugins to get the following screens. For tests, my app only has the Dropdown component and nothing else.

And it seems the @react-aria libs is imported multiple times :

![Capture d’écran 2022-07-10 075621](https://user-images.githubusercontent.com/24851412/178133239-d9b1caee-84ac-48a8-9258-ac4d49e1de26.png)

But I may be wrong, I'm not particulary familiar with these manipulations....

**Comment by TIMMLOPK** [comment]
Hope https://github.com/nextui-org/nextui/pull/663 would help
